roles/rsyslog/templates/output_files.j2

```diff
@@ -1,13 +1,7 @@
 {% if files_item.path is defined %}
-{%   if files_item.exclude is defined %}
 ruleset(name="{{ files_item.name }}") {
-    {{ files_item.facility | d('*') }}.{{ files_item.severity | d('*') }};{{ files_item.exclude }} {{ files_item.path }}
+    {{ files_item.facility | d('*') }}.{{ files_item.severity | d('*') }}{{ files_item.exclude | d() }} {{ files_item.path }}
```

Comment:
if `exclude` becomes a true list, use the join filter: https://jinja.palletsprojects.com/en/2.9.x/templates/#join